app/components/Accounts.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+import { X } from "lucide-react";
+import { useRouter } from "next/navigation";
+
+const Accounts = () => {
+  const [showPopup, setShowPopup] = useState(false);
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
```

Comment:
add typescript type

example: 
const [showPopup, setShowPopup] = useState<boolean>(false);

more examples: string, string[], int, bool etc. 
use correct type for corresponding useState variable